brightchain-lib/src/lib/documents/member/memberDocument.ts

```
@@ -0,0 +1,138 @@
+  public abstract toPublicCBL(): Promise<SerializableBuffer>;
+
+  /**
+   * Convert private data to CBL
+   */
+  public abstract toPrivateCBL(): Promise<SerializableBuffer>;
+
+  /**
+   * Create from CBLs
+   */
+  public static async createFromCBLs(
+    publicCBL: SerializableBuffer,
```

ESLint

[error] 'publicCBL' is defined but never used.

(@typescript-eslint/no-unused-vars)